docs/reference/search/request/preference.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[search-request-preference]]
 === Preference
 
-Controls a `preference` of which shard replicas to execute the search
-request on. By default, the operation is randomized between the shard
-replicas.
+Controls a `preference` of which shard copies on which to execute the 
+search. By default, the operation is randomized among the primary 
```

Comment:
I think we can also use the terminology "share copies" that you used in the first sentence, I like it.